lib/vehicle.rb

```diff
@@ -1,18 +1,18 @@
 require 'date'
 
 class Vehicle
-  attr_reader :vin,
-              :year,
-              :make,
-              :model,
-              :engine
+  attr_reader :vin, :year, :make, :model, :engine
+              
+  attr_accessor :registration_date
```

Comment:
I'm getting several errors when I run your specs:
```
NoMethodError:
       undefined method `plate_type=' for #<Vehicle:0x00000001510f28f8>
```
Adding `:plate_type` to your accessors should fix that. I'm curious if you also saw this error before submitting or if there was a different way you thought this was being handled?

```suggestion
  attr_accessor :registration_date, :plate_type
```